catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -1124,6 +1133,37 @@ private FilterCapabilities buildFilterCapabilities() {
     return filterCapabilities;
   }
 
+  private List<String> getQueryFilterTransformerTypeNames() {
+    return Stream.of(getBundle())
+        .map(Bundle::getBundleContext)
+        .map(
+            new Function<BundleContext, Collection<ServiceReference<QueryFilterTransformer>>>() {
```

Comment:
nit : extracting this to its own method might be better from a readability standpoint

Reply:
done